operations/node_management/Rebuild_NCNs/Prepare_Storage_Nodes.md

```diff
@@ -45,11 +45,67 @@ If rebuilding `ncn-s001`, it is critical that the `storage-ceph-cloudinit.sh` ha
 
 ## Procedure
 
+Upload ceph container images into nexus.
```

Comment:
we should be specific and tell then what to name it and where to put it.    we can also make this a script file and put in the docs-csm/scripts dir and just have copy out, chmod, and a run example.